docs/app/pages/components/components-sections/input-controls/input-dropdown/input-dropdown.component.html

```diff
@@ -96,6 +103,9 @@
     <tr uxd-api-property name="aria-label" type="string">
         Aria label of the filter field. If not specified, the placeholder will be used.
     </tr>
+    <tr uxd-api-property name="dropdownOpen" type="boolean">
+        The status of the typeahead dropdown.
```

Comment:
I think this description could be a bit more clear. We have lots of other components that have `open` inputs, consider something more like them, eg:

If set to `true` this will programmatically show the typeahead dropdown